src/services/checkov/executor.ts

```diff
@@ -62,6 +62,11 @@ export class CheckovExecutor {
             return;
         }
 
+        if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 1 && !targetFiles) {
+            vscode.window.showWarningMessage('Full scan is only supported when working with a single VS Code workspace');
```

Comment:
When a user clicks the scan button we use vscode.workspace.workspaceFolders[0]. To avoid use errors (and PCSUPS) I'm showing this warning, and if by chance customers will require it we can develop this feature.